src/display/editor/tools.js

```diff
@@ -1442,12 +1450,29 @@ class AnnotationEditorUIManager {
     }
   }
 
+  /**
+   * Update the UI of the active editor.
+   * @param {AnnotationEditor} editor
+   */
+  updateUI(editor) {
+    if (this.#lastSelectedEditor === editor) {
+      this.#dispatchUpdateUI(editor.propertiesToUpdate);
+    }
+  }
+
   /**
    * Add or remove an editor the current selection.
    * @param {AnnotationEditor} editor
    */
   toggleSelected(editor) {
     if (this.#selectedEditors.has(editor)) {
+      if (this.#lastSelectedEditor === editor) {
+        let ed = null;
+        for (ed of this.#selectedEditors) {
+          // Iterate to get the last element.
+        }
+        this.#lastSelectedEditor = ed;
+      }
```

Comment:
Sorry, but does this actually work as intended?

Should we not remove the `editor` from the set of selected ones *first*, see the line just below, since otherwise it seems to me that we risk updating the "last selected" editor to the one we're about to *remove* from the set of selected ones?